apps/worker/src/app/workflow/usecases/send-message/send-message-chat.usecase.ts

```diff
@@ -397,14 +397,18 @@ export class SendMessageChat extends SendMessageBase {
         ...(command.overrides[integration?.channel] || {}),
         ...(command.overrides[integration?.providerId] || {}),
       };
-
-      const result = await chatHandler.send({
-        phoneNumber,
-        customData: overrides,
-        webhookUrl: chatWebhookUrl,
-        channel: channelSpecification,
-        ...(bridgeContent?.content ? bridgeContent : { content }),
-      });
+      const bridgeProviderData = command.bridgeData?.providers?.[integration.providerId] || {};
+
+      const result = await chatHandler.send(
+        {
+          phoneNumber,
+          customData: overrides,
+          webhookUrl: chatWebhookUrl,
+          channel: channelSpecification,
+          ...(bridgeContent?.content ? bridgeContent : { content }),
+        },
+        bridgeProviderData
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Why are we adding a new positional argument here? Can we add `bridgeProviderData` as an optional property on the first positional argument instead?

New positional arguments complicate extension, usually required to pass a default object like `{}` to make Typescript whilst adding redundant code. We should prefer to extend objects instead.‏